lang/c++/api/DataFile.hh

```diff
@@ -75,11 +75,11 @@ class AVRO_DECL DataFileWriterBase : boost::noncopyable {
     Metadata metadata_;
     int64_t lastSync_;
 
-    static std::unique_ptr<OutputStream> makeStream(const char* filename);
+    static std::unique_ptr<OutputStream> makeStream(const char *filename);
```

Comment:
same comment as on prior PRs we should ensure this type of style change is discussed and agreed upon on the ML.  Also, having a GHA with formatter that enforces this one way or another would be nice.

Reply:
From the next installment, I'll precede the actual PR with a format-change PR.